src/GoogleAuth/getGoogleCredentials.ts

```diff
@@ -20,7 +20,8 @@ export const getGoogleCredentials = async () =>
   withTimeout(async () => {
     const torus = new CustomAuth({
       web3AuthClientId: WEB3_AUTH_CLIENT_ID,
-      baseUrl: "https://umamiwallet.com/auth/v2/",
+      // TODO: set to production
```

Comment:
Is this bit required for the change?
If so, is there a ticket for this TODO?

Reply:
Sorry, forgot to mark it as draft. The pr is not ready to be shipped yet